### PR TITLE
Updating news file for 2.3.0

### PR DESCRIPTION
**Pull request by waldbauer-certat**
Updating news file for 2.3.0

- [x] ShadowServer updates
- [x] Sieve Expert updates
- [x] Kafka Collector
- [x] IntelMQ-API replaces IntelMQ-Manager

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -32,13 +32,32 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
+- `intelmqctl list queues` is now able to count events in a queue with the `--sum`, `--count` or simple `-s` flag.
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### ShadowServer Bots
+We've updated ShadowServer Collector & Parser for all reports as of now (2/11/2021).
```
Comment (schacht-certat) [inline NEWS.md:44]: I have no idea what that sentence means, can you rephrase? And given thats a `NEWS` file with changes that relate to a specific release, I think the date is not needed.

File: NEWS.md [hunk 2]
```diff
@@ -32,13 +32,32 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
+
+#### Sieve Expert
+We've updated the sieve-expert, which has two nice options now. More details can be found in our documentation.
```
Comment (schacht-certat) [inline NEWS.md:47]: Options are not `nice`

Comment (schacht-certat) [reply NEWS.md:47]: The 'more details' sentence should come after the examples.

File: NEWS.md [hunk 3]
```diff
@@ -32,13 +32,32 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
+
+1. Basic math directly in sieve, only DateTimes currently!
```
Comment (schacht-certat) [inline NEWS.md:49]: There is a verb missing or two and the exclamation mark should be replaced by a colon

File: NEWS.md [hunk 4]
```diff
@@ -32,13 +32,32 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
+```
+add time.observation += '1 hour'
+add time.observation -= '10 hours'
+```
+2. Actions without an `if` statement, just write the following into a `.sieve` file
```
Comment (schacht-certat) [inline NEWS.md:54]: Actions **can now be used** without...?

File: NEWS.md [hunk 5]
```diff
@@ -47,6 +66,12 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+In `2.3.0` release `IntelMQ-Manager` is getting replaced by `IntelMQ-API`, only frontend files from `IntelMQ-Manager` are used. So you have to reinstall the `IntelMQ-Manager` & install the `IntelMQ-API`, in order to update your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```
Comment (schacht-certat) [inline NEWS.md:70]: This is part of the news entry for the 2.3.0 release, I don't think its necessary to repeat the version here. `only frontend files from IntelMQ-Manager` are used should probable read: "The IntelMQ-Manager release only contains the files for the web frontend". You should not have to reinstall anything, the upgrade path should handle the move to intelmq-api (thats why intelmq-manager depends on intelmq-api).

File: NEWS.md [hunk 6]
```diff
@@ -47,6 +66,12 @@ The feed template for the URLHaus feed contained a spelling error:
...
+
+But why is there a change now?
+Well, the `IntelMQ-API` was made as part of the `CEF5` project & is now the core part for `frontends` to communicate with. `IntelMQ-API` communicates with the `IntelMQ` cluster itself and can setup bots & manage configurations, like you know and you've done with `IntelMQ-Manager` before. But now you're able to create configurations/start bots with a simple API call.
+
```
Comment (schacht-certat) [inline NEWS.md:74]: Not sure if that paragraph makes sense in a `NEWS` file.

**Review by schacht-certat — CHANGES_REQUESTED** [review]
The changes introduce writing in first person, which in the rest of the document is not used. I'm not sure it makes sense to change writing style and first person is also not commonly used in technical writing. Was this intentional?

File: NEWS.md [hunk 7]
```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
   intelmq.bots.experts.tor_nodes.expert --update-database
   ```
   The provided shell scripts use these new commands, however they are now deprecated and will be removed in version 3.0.
+- `intelmqctl list queues` is now able to count events in a queue with the `--sum`, `--count` or simple `-s` flag.
```
Comment (wagner-certat) [inline NEWS.md:35]: The change is not that `intelmqctl list queues` can "count events in a queue" (it had that functionality already), but that it can give a global sum over all queues.

File: NEWS.md [hunk 8]
```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### ShadowServer Bots
+ShadowServer Collector & Parser reworked. CSV and JSON reports are now parseable.
```
Comment (wagner-certat) [inline NEWS.md:44]: Should better indicate that the JSON support is the new thing here, additionally to the existing CSV parser.
And the Collector and the parser have not been reworked. We added the JSON parsing functionality and a new collector for the Shadowserver Reports API.
Please also refer to the Bots documentation for more details.

File: NEWS.md [hunk 9]
```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
   The provided shell scripts use these new commands, however they are now deprecated and will be removed in version 3.0.
+- `intelmqctl list queues` is now able to count events in a queue with the `--sum`, `--count` or simple `-s` flag.
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### ShadowServer Bots
```
Comment (wagner-certat) [inline NEWS.md:43]: I think they call themselves "Shadowserver" (I see that this needs to be fixed in other places in docs and code as well, not a pressing issue though)

File: NEWS.md [hunk 10]
```diff
@@ -47,6 +68,9 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+`IntelMQ-Manager` is getting replaced by `IntelMQ-API`. The `IntelMQ-Manager` release only contains the files for web frontend. Install the `IntelMQ-API`, in order to upgrade your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```
Comment (wagner-certat) [inline NEWS.md:72]: I wouldn't say that the Manager was replaced by the API, but the Manager's PHP-based backend was replaced by the newly developed Python-based IntelMQ API.

Please also add, that those who use the deb/rpm packages automatically receive the IntelMQ API via package dependencies upon upgrade.

**Review by wagner-certat — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 11]
```diff
@@ -32,13 +32,34 @@ CentOS 7 (with EPEL) provides both Python 3.4 and Python 3.6. If IntelMQ was ins
...
+- `intelmqctl list queues` is now able to sum events over all queues with the `--sum`, `--count` or simple `-s` flag.
 
 #### XMPP Bots
 Both the XMPP output bot and the XMPP collector bot are deprecated.
 The bots need to be migrate to another XMPP library (see [#1614](https://github.com/certtools/intelmq/issues/1614) for details) and a survey on the mailing listed revealed no users.
 If you depend on this bot, please reach out to us via the mailing list or GitHub.
 The bots are logging a deprecation warning now and the current plan is to remove them in IntelMQ version 3.0.
 
+#### Shadowserver Bots
+Shadowserver Collector is now able to collect data from [Shadowserver Reports API](https://intelmq.readthedocs.io/en/latest/user/bots.html#shadowserver-reports-api). Moreover Shadowserver Parser now supports JSON.
```
Comment (schacht-certat) [inline NEWS.md:44]: **The** Shadowserver collector ... **the** Shadowserver Reports API... **the** Shadowserver parser now supports **the JSON format used by the Reports API**

File: NEWS.md [hunk 12]
```diff
@@ -47,6 +68,11 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+`IntelMQ-Manager` PHP backend is getting replaced by newly developed `IntelMQ-API`. The `IntelMQ-Manager` release only contains the files for web frontend. Install the `IntelMQ-API`, in order to upgrade your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```
Comment (schacht-certat) [inline NEWS.md:72]: **the** newly developed `IntelMQ-API`... for **the** web frontend...

And I would it phrase like this: The `intelmq-manager` packages now depend or recommend the installation of the `intelmq-api` package, therefore a normal upgrade should pull in the API. Follow the instructions in our documentation to configure the API. It uses its own authentication backend now, how to setup users is described in the documentation.

File: NEWS.md [hunk 13]
```diff
@@ -47,6 +68,11 @@ The feed template for the URLHaus feed contained a spelling error:
...
+
+**NOTE** If you're using deb/rpm packages, `IntelMQ-API` will automatically be installed if you're upgrading `IntelMQ-Manager`.
+
```
Comment (schacht-certat) [inline NEWS.md:75]: Thats also true for the Python packages, but the rephrasing above would include this information.

**Review by schacht-certat — CHANGES_REQUESTED** [review]
See inline comments

File: NEWS.md (hunk 13 above)
Comment (wagner-certat) [reply NEWS.md:75]: For the manager, there hasn't been a python package before.

**Review by wagner-certat — APPROVED** [review]
